docs/kubernetes/deploy.md

```diff
@@ -0,0 +1,132 @@
+# Deploy a Kubernetes Cluster
+
+## Install Pre-requisites
+
+All the commands in this guide require both the Azure CLI and `acs-engine`. Follow the [installation instructions to download acs-engine before continuing](../acsengine.md#install-acs-engine) or [compile from source](../acsengine.md#build-from-source).
+
+For installation instructions see [the Azure CLI GitHub repository](https://github.com/Azure/azure-cli#installation) for the latest release.
+
+## Overview
+
+`acs-engine` reads a cluster definition which describes the size, shape, and configuration of your cluster. This guide takes the default configuration of one master and two linux agents. If you would like to change the configuration, edit `examples/kubernetes.json` before continuing.
+
+The `acs-engine deploy` command automates creation of a Service Principal, Resource Group and SSH key for your cluster. If operators need more control or are are intersted in the individual steps see the ["Long Way" section below](#the-long-way).
```

Comment:
`acs-engine deploy` doesn't generate an SSH key yet. It's the only piece of config users have to paste into their cluster definition. On the docket to fix this, but the instructions should match current functionality.